tests/framework/extensions/clusters/clusters.go

```diff
@@ -275,18 +272,30 @@ func CreateRKE1Cluster(client *rancher.Client, rke1Cluster *management.Cluster)
 	}
 
 	client.Session.RegisterCleanupFunc(func() error {
-		err := client.Management.Cluster.Delete(rke1Cluster)
+		adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
```

Comment:
Why are you using the admin client to clean up the cluster? What user created it, should be able to clean it up. This also to ensure proper cluster role permissions are working as expected.

Reply:
I can revert that back no problem. I saw that RKE2/K3s did the same thing, so I was more so having parity with that.

Reply:
Really? Nevermind, sorry Markus I just checked. I'm fine with this being changed. I should have checked beforehand.